wpn_optic2/Stringtable.csv

```diff
@@ -0,0 +1,14 @@
+﻿////Multiplicity: <br/>Range: <br/>Modes: 
+////Кратность: <br/>Дальность: <br/>Режимы: 
+///////////////
+Language, "Original"
+
+STR_pkas_name,"PK-As"
+STR_pkas_desc,"1x Russian Collimator optic. Fits RMP-5 rails."
+STR_pk01_name,"PK-As"
+
+STR_pkasDrob_name,"PK-As"
+STR_pkasDrob_desc,"1x Russian Collimator optic. Fits RMP-5 rails."
+
+STR_p29_name,"1P29"
```

Comment:
Should avoid giving attachments names that start with numbers, or they'll be selected by default.